addons/survey/static/src/scss/survey_survey_views.scss

```diff
@@ -124,33 +124,22 @@
 }
 
 /* Style of the tiles allowing the user to load a sample survey. */
-.o_survey_sample_container  {
-    margin-top: .5rem;
-}
+.o_survey_sample_container {
+    pointer-events: auto;
 
-.o_survey_sample_tile {
-    max-width: 150px;
-    padding: 1rem;
-    font-size: .75rem;
-    .o_survey_sample_tile_cover {
-        display: none;
-        overflow-y: auto;
-        cursor: pointer;
-    }
-    &:hover {
-        .o_survey_sample_tile_cover {
-            display: flex;
-            flex-direction: column;
-            background: rgba(0, 0, 0, 0.9);
-            &::before, &::after {
-                content: '';
-            }
-            &::before {
-                margin-top: auto;
-            }
-            &::after {
-                margin-bottom: auto;
+    .survey_card {
+        &, * {
+            transition: all .15s;
```

Comment:
Do we really need a transition? .15s is really short and honestly barely noticeable. 

I also don't like that we apply it on every node (* rule) and on every rule ("all"), if we really want to keep it then we should be a little more precise on where and how it applies.

Reply:
It works fine, removed transition if not needed then :+1: